### PR TITLE
Fix for ARM Linuxbrew for Raspberry Pi

### PR DESCRIPTION
**Pull request by AlexisMeza**
Resolved issues on Raspberry Pi with glibc

### CODE REVIEW

File: Library/Homebrew/cmd/install.rb [hunk 1]
```diff
@@ -194,7 +194,8 @@ def check_ld_so_symlink
     ld_so = HOMEBREW_PREFIX/"lib/ld.so"
     return if ld_so.readable?
     glibc = Formula["glibc"]
-    interpreter = glibc && glibc.installed? ? glibc.lib/"ld-linux-x86-64.so.2" : "/lib64/ld-linux-x86-64.so.2"
+    sys_interpreter = File.exist?("/lib64/ld-linux-x86-64.so.2") ? "/lib64/ld-linux-x86-64.so.2" : "/lib/ld-linux.so.3"
```
Comment (sjackman) [inline Library/Homebrew/cmd/install.rb]: Can you change this to: (untested)
```ruby
sys_interpreter = ["/lib64/ld-linux-x86-64.so.2", "/lib/ld-linux.so.3", "/lib/ld-linux.so.2"].find do |s|
        Pathname.new(s).executable?
end
```

**Comment by sjackman** [comment]
Thanks for the patch, Alexis. It's looking good.

**Comment by AlexisMeza** [comment]
I tested your change and it worked.

**Comment by sjackman** [comment]
Merged. Thanks, Alexis!

**Pull request closed without merging**